README.md

```diff
@@ -1,102 +1,367 @@
-
-This necessitates 2 key capabilities of the supergraph platform to create a consumer-first, agile culture:
-1. API consumption, API schema &amp; portal analytics: A supergraph is analogous to a marketplace and needs to provide the marketplace owners and producers with insights to help improve the marketplace for the consumers.
-2. Ecosystem integrations: The supergraph platform should be able to integrate with existing communication and catalog tools, in particular to help understand _unmet_ demand of API consumers.
-
-**This closes the loop and allows the supergraph platform to create a virtuous cycle of success for producers and consumers**.
-
-## Supergraph reference architecture
-
-### CI/CD and build system (control plane)
-The control plane of the supergraph is critical to help domain owners [connect their domains](#i.-connect-domains) to the supergraph.
-
-There are 3 components in the control plane of the supergraph
-1. The domain itself
-2. The subgraph
-3. The supergraph
-
-<img width="800" alt="Supergraph control plane components" src="https://github.com/hasura/supergraph-io/assets/131160/e661efd8-9d0f-4340-a5a1-a119e9fc87ee">
-
-The control plane should define the following SDLC to help keep the supergraph in sync with the domain as the underlying domain changes.
-
-<img width="800" alt="Supergraph CI/CD" src="https://github.com/hasura/supergraph-io/assets/131160/e6fec5a3-e3da-447e-9ac3-5dc0ceef66d9">
-
-### Distributed data plane
-The supergraph data plane is critical to enable high performance access to upstream domains so that API producers can maintain their domain without hidden future maintenance costs:
-
-<img  width="800" alt="Supergraph data plane" src="https://github.com/hasura/supergraph-io/assets/131160/c6e1de9b-fe8f-4f9e-8503-7f655b02d9a9">
-
-### Supergraph API schema
-
-**Standardization**
-
-A supergraph API schema should create standardized conventions on the following:
-- Queryable models vs Commands
-  - Models are collections of data that can be queried in standardized ways
-  - Commands are methods that map to particular pieces of business logic that might return references to other commands or models
-- Standardized conventions on queryable model: While each model might only expose some ways of querying it, the syntax and conventions for standard query operations should be standardized
-  - Joins
-  - Filtering
-  - Pagination
-  - Sorting
-  - Aggregations
-
-**Composability**
+![ConnectConsumeDiscoverERD.png](assets/ConnectConsumeDiscoverERD.png)
+
+### Connect Domains
+As an API producer, you should easily and seamlessly connect your data domains to the platform. The Supergraph strategy 
+and architecture addresses common challenges and reluctance from domain owners by ensuring:
+
+1. **Subgraph Connector CI/CD**: Keeps the API contract in sync with minimal overhead for the domain owner.
```

Comment:
As we haven't introduced the concept of a connector, I think it's extra overhead to mention it here. Above, we're saying that a user should be able to connect seamlessly — this means the connector — once it's deployed — is forgotten about. What they then care about modifying and updating is the subgraph itself and its performance.

```suggestion
1. **Subgraph CI/CD**: Keeps the API contract in sync with minimal overhead for the domain owner.
```